### PR TITLE
Only render div if the user is able to connect another auth

### PR DESCRIPTION
**Pull request by Zhao-Andy**
## What type of PR is this? (check all applicable)
- [x] Bug Fix

## Description
Fixes a small bug where a blank `<div>` is rendered for users with more than 1 authentication identity.

# Before:
Screen Shot 2020-06-17 at 11 11 24 AM

# After:

Screen Shot 2020-06-17 at 11 08 52 AM

### CODE REVIEW

File: app/views/users/_profile.html.erb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <%= javascript_packs_with_chunks_tag "colorPreview", "validateFileInputs", defer: true %>
 
-<div class="crayons-card mb-6 p-5">
-  <%= render "users/additional_authentication" %>
-</div>
+<% if @user.identities.size < 2 %>
```
Comment (rhymes) [inline app/views/users/_profile.html.erb:3]: didn't know we added this too!

can you use `if user.identities.count < SiteConfig.authentication_providers.count` ? I shall add a custom method soon

Comment (Zhao-Andy) [reply app/views/users/_profile.html.erb:3]: Ohh yeah that's great 👍

Comment (Zhao-Andy) [reply app/views/users/_profile.html.erb:3]: I'm going to stick with `@user.identities.size` or `.count` unless you think otherwise?

Comment (rhymes) [reply app/views/users/_profile.html.erb:3]: no that's fine!

Comment (rhymes) [reply app/views/users/_profile.html.erb:3]: BTW in `/account` there's the same problem so it's probably easier to add it to the partial itself

**Comment by Zhao-Andy** [comment]
Fixed for `/account` too, thanks @rhymes!

# Before:

Screen Shot 2020-06-17 at 11 31 56 AM

# After:

Screen Shot 2020-06-17 at 11 32 02 AM

**Pull request merged**